components/Buttons/src/MDCFloatingButton.h

```diff
@@ -74,7 +74,16 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonImageLocation) {
 
 /**
  The mode of the floating button can either be .normal (a circle) or .expanded (a pill-shaped
- rounded rectangle).
+ rounded rectangle). In the @c .normal mode, the button should have either an image or a title,
```

Comment:
The "shoulds" here make me nervous - is this something the client needs to be cautious not to do? Can we enforce it somehow?

Reply:
What happens if both are provided?

Reply:
It's based on Material Design guidelines. I don't think we should force an either-or in this case. It's possible that the client is changing the minimum sizes for `.normal` and can actually fit both. Alternatively, they may be adjusting insets so that the image and text align with one another.

Here's with a standard image + title ("Plus")
<img width="83" alt="fab-image title" src="https://user-images.githubusercontent.com/1753199/33588723-57bc2b02-d942-11e7-8c2d-a4c65f5af56d.png">

Here's one where the insets are adjusted to (poorly) align the image/text. It does violate the design guidelines, but I don't think there's much to be gained by making it hard to achieve.

<img width="92" alt="fab-image title-insets" src="https://user-images.githubusercontent.com/1753199/33588796-b50bbf2a-d942-11e7-8ce4-01bcb187d302.png">

A more creative button might have the image and text go well together, unicode has a lot of cool glyphs!

<img width="114" alt="fab-image title-glyph" src="https://user-images.githubusercontent.com/1753199/33589129-c34487f0-d944-11e7-8bd9-45f11d0a050b.png">